### PR TITLE
Description in TestCase

### PR DESCRIPTION
**Pull request by singhsarab**
Make Description, CssIteration, WorkItem, CssProjectStructure available in properties in the TestCase object.
Fixes : #529

### CODE REVIEW

File: src/Adapter/MSTest.CoreAdapter/Helpers/ReflectHelper.cs [hunk 1]
```diff
@@ -414,6 +414,24 @@ internal virtual Attribute[] GetCustomAttributes(MemberInfo attributeProvider, T
...
+        /// Gets the first custom attribute of the provided type on a memberInfo
+        /// </summary>
+        /// <param name="attributeProvider"> The member to reflect on. </param>
+        /// <param name="type"> The attribute type. </param>
+        /// <returns>Attribute defined.</returns>
+        internal virtual Attribute GetCustomAttribute(MemberInfo attributeProvider, Type type)
+        {
+            var attribute = GetCustomAttributes(attributeProvider, type, true);
+
+            if (attribute == null || attribute.Length != 1)
```
Comment (jayaranigarg) [inline src/Adapter/MSTest.CoreAdapter/Helpers/ReflectHelper.cs:427]: What if we have multiple workitem attributes? Do we want to silently ignore/not fill them in property bag?

Comment (singhsarab) [reply src/Adapter/MSTest.CoreAdapter/Helpers/ReflectHelper.cs:427]: That's a valid concern. I just followed the same convention that was bein followed for Priority and Owner. https://github.com/Microsoft/testfx/blob/8384164a3187cd586134bfc230c337671300f68d/src/Adapter/MSTest.CoreAdapter/Helpers/ReflectHelper.cs#L599

Comment (jayaranigarg) [reply src/Adapter/MSTest.CoreAdapter/Helpers/ReflectHelper.cs:427]: We cannot have multiple owner and priority attributes for a method as per framework restrictions. https://github.com/Microsoft/testfx/blob/master/src/TestFramework/MSTest.Core/Attributes/VSTestAttributes.cs#L188
But we can indeed have multiple workitem attributes. https://github.com/Microsoft/testfx/blob/master/src/TestFramework/MSTest.Core/Attributes/VSTestAttributes.cs#L297

Comment (singhsarab) [reply src/Adapter/MSTest.CoreAdapter/Helpers/ReflectHelper.cs:427]: Good catch. Then, we'd ideally want to make workitem property an array of workitems.

**Comment by singhsarab** [comment]
@dotnet-bot test this please

**Pull request merged**